modules/azure-gaming/connected-resources/modules/ContainerService/managedClusters.bicep

```diff
@@ -0,0 +1,104 @@
+@description('Deployment Location')
+param location string
+param name string = 'k8-cluster'
+param agentPoolCount int = 3
+param agentPoolName string = 'k8agent'
+param vmSize string = 'Standard_L16s_v2'
+param nodeLabels string = 'horde-storage'
+param assignRole bool = false
+param dnsPrefix string = 'k8-${take(uniqueString(name), 5)}'
+param kubernetesVersion string = '1.22.11'
```

Comment:
The portal default for k8s version is `1.23.12`
1.22.11 is currently the oldest deployable version
![image](https://user-images.githubusercontent.com/17914476/203259378-15fc16b0-07f8-4d73-81f3-11f1a0288af2.png)

Reply:
Should this param not have a default value? it seems like the supported k8s version gets updated very frequently. Maybe the param description can have a link for the latest supported versions?

Reply:
> > I do see some similarity here with what I previously proposed in #61 (and was rejected).
> 
> @Gordonby - I think back in April we were hoping we would be able to onboard CARML modules more easily. We are more open now to onboarding "base resource" modules. From #61, it looks like that modules was explicitly intended as a sample, which is what quickstarts if for. If it's not intended to be used as a general purpose utility, then it should live here.

I think back then i was concerned it didn't align to a specific azure service so was trying to work out what the right directory to put it in would be.

Reply:
and.... '1.22.11' disappeared from East US, so it's time to update again!

Reply:
```suggestion
param kubernetesVersion string = '1.23.8'
```